src/js/components/RadioButtonGroup/radiobuttongroup.stories.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, RadioButtonGroup } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+class SimpleRadioButtonGroup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: props.value };
+  }
+
+  onChange = event => this.setState({ value: event.target.value });
+
+  render() {
+    const { value } = this.state;
+    return (
+      <Grommet theme={grommet}>
+        <Box align="center" pad="large">
+          <RadioButtonGroup
+            name="radio"
+            options={[
+              { label: 'Choice 1', value: 'c1' },
+              { label: 'Choice 2', value: 'c2' },
+              { label: 'Choice 3', value: 'c3' },
```

Comment:
the buttons are not really working on the example, and the focus is always set on the first choice (even when clicking on others).